### PR TITLE
feat: Add 'Change Language' screen

### LINKED ISSUE #21: Add "Change language" screen

### PR DESCRIPTION
**Pull request by NelsonPRSousa**
Closes #21

### CODE REVIEW

File: src/components/screens/settings/components/language-list-item/style.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
```
Comment (joaopedrodcf) [inline src/components/screens/settings/components/language-list-item/style.ts]: We are using styled components per styling this is a different way to styling from the one we been using.

Should be easy to change you just need to create styled components per style you have in here an things like active can come as a prop from the component.

File: src/components/screens/settings/components/language-selector/LanguageSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import LanguageListItem from '../language-list-item/LanguageListItem';
+
+import languages from '../../../../../mocks/languages';
+
+// TODO: this should be some kind of state (we need persistency here, too -> when the user closes and reopens the app)
+const currentLocale = 'pt';
+
+class LanguageSelectorScreen extends React.PureComponent<{}, {}> {
+    render(): JSX.Element {
```
Comment (joaopedrodcf) [inline src/components/screens/settings/components/language-selector/LanguageSelector.tsx]: Do you really need to infer the type JSX.Element as a return of this I think as we extends React.PureComponent we knows it returns JSX no?

File: src/components/screens/settings/components/language-selector/LanguageSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        return (
+            <View style={{ marginTop: 15 }}>
```
Comment (joaopedrodcf) [inline src/components/screens/settings/components/language-selector/LanguageSelector.tsx]: We should avoid inline styles, is gonna rerender this component. As a workaround you can create a variable style outside of the component class and do in here
```
style={style}
```

File: src/components/screens/settings/components/language-selector/LanguageSelector.tsx (hunk 2 above)
Comment (NelsonPRSousa) [reply src/components/screens/settings/components/language-selector/LanguageSelector.tsx]: Initially I didn't have it, but our current TSLint configuration seems to cause a warning regarding this:

image

What do you suggest?

Comment (joaopedrodcf) [reply src/components/screens/settings/components/language-selector/LanguageSelector.tsx]: kill it with fire 🗡 

The problem is this being a class normally we being using function components until now, we could use hooks but maybe has a bit learning curve.
Lets put the type back at the render sorry 👍

File: src/components/screens/settings/components/language-list-item/style.ts (hunk 1 above)
Comment (NelsonPRSousa) [reply src/components/screens/settings/components/language-list-item/style.ts]: Thanks for the heads up! Fixed accordingly 👍

File: src/components/screens/settings/components/language-selector/LanguageSelector.tsx (hunk 3 above)
Comment (NelsonPRSousa) [reply src/components/screens/settings/components/language-selector/LanguageSelector.tsx]: Thanks for the heads up! Fixed accordingly 👍

File: src/components/screens/settings/components/language-selector/LanguageSelector.tsx (hunk 2 above)
Comment (NelsonPRSousa) [reply src/components/screens/settings/components/language-selector/LanguageSelector.tsx]: I am now using FC, avoiding that type!

Can you please review this pull request?

**Pull request merged**